client/elements/sc-page-search.html

```diff
@@ -592,9 +588,44 @@ <h2 class="search-result-title">[[_calculateTitle(item)]]</h2>
               if (!item.heading.title) {
                   return '';
               } else {
-                  return item.acronym ? item.acronym : item.uid;
+                  return item.acronym ? `${this._convertAcronym(item.acronym)} — ${item.author}` : `${this._transformId(item.uid)} — ${item.author}`;
+              }
+          }
+
+          _convertAcronym(acronym) {
+              if (acronym.match(/\/\//)) {
+                  return acronym.replace(/\/\//, " (") + ")";
+              } else {
+                  return acronym;
+              }
+          }
+
+          _transformId(rootId) {
+              if (!rootId || !this.expansionReturns) {
+                  return '';
+              }
+              const expansionData = this.expansionReturns;
+              let scAcronym = '';
+              const uidParts = rootId.split('-');
+              let tail = '';
+              try {
+                  uidParts.forEach(item => {
+                      if (!expansionData[0][item]) {
+                          const tailMatch = item.match(/\d+.*/g);
+                          if (tailMatch) tail = tailMatch[0] + '–';
+                          const itemMatch = item.match(/[a-z]*/g);
+                          if (itemMatch) item = itemMatch[0];
+                      }
+                      if (item && expansionData[0][item]) {
+                          scAcronym += `${expansionData[0][item][0]} ${tail}`
+                      } else {
+                          scAcronym += tail;
+                      }
+                  });
+                  return scAcronym.replace(/–\s*$/, '');
+              } catch (e) {
+                  console.error(e);
```

Comment:
right now, if there's an error somewhere in the `try` block, this function will return `undefined` and that's going to be displayed in the UI.